src/sentry/monitors/logic/mark_failed.py

```diff
@@ -304,6 +326,41 @@ def create_issue_platform_occurrence(
     )
 
 
+HUMAN_FAILURE_STATUS_MAP: Mapping[int, str] = {
```

Comment:
i don't think this type is correct? `CheckInStatus` is an enum not an `IntEnum`?

Reply:
It's not an enum

Reply:
https://github.com/getsentry/sentry/blob/ac80048be29f718d4caf7000034be7ac19f0e8f3/src/sentry/monitors/models.py#L125